ci/pip-requirements.txt

```diff
@@ -1,5 +1,8 @@
-# Always track ixmp master
-git+git://github.com/iiasa/ixmp.git#egg=ixmp
+## Always track ixmp master
+#git+git://github.com/iiasa/ixmp.git@#egg=ixmp
+
+# Branch for https://github.com/iiasa/ixmp/pull/254
+git+git://github.com/khaeru/ixmp.git@post-2.0-cleanup#egg=ixmp
```

Comment:
Note that this change is only to test against iiasa/ixmp#254 before that PR is merged. The steps will be:

1. Merge the ixmp PR.
2. Revert the changes to this file.
3. Confirm that the checks still pass for this PR.
4. Merge this PR.